webui/src/components/editrecord.jsx

```diff
@@ -99,6 +99,9 @@ const EditRecord = React.createClass({
             errors: {},
             dirty: false,
             waitingForServer: false,
+            readOnly: false,
+            checkboxValue: null,
+            revoking:false,
```

Comment:
would help to add another word to this name, about what is being revoked -- I'm missing the context now

Reply:
revoking_submitted_draft? Or revoking_submitted_for_review?

Reply:
yes,  `revokeSubmitted` or even `unsubmit`, your choice here